src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx

```diff
@@ -0,0 +1,63 @@
+---
+subject: Docs
+releaseDate: '2024-04-26'
+version: '4-26-2024'
+---
+
+<Callout variant="important">
+We're trying out release notes to let you know what's changing in our docs each week. This is an experiment, so [let us know via GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose) if you find them helpful!
+</Callout>
+
+### New docs
+
+* Our new [NVIDIA Triton integration](/docs/infrastructure/host-integrations/host-integrations-list/nvidia-triton-integration) monitors the deployment and management of AI models in production environments via the infrastructure agent.
+* Our infrastructure agent Azure monitor integration now supports [Azure Recovery Services Vaults](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-recovery-services-vaults-monitoring-integration).
+
+### Major changes
+
+* A brand new Docs site home page: [check it out](https://docs.newrelic.com/)!
```

Comment:
Can we add a bit about the logic of why? You can crib from my announcement message in the help channel